pkg/models/models.go

```diff
@@ -5,26 +5,38 @@ import (
 )
 
 type User struct {
-	ID                    string    `json:"id" yaml:"id"`
-	CreatedAt             time.Time `json:"createdAt" yaml:"createdAt"`
-	Email                 string    `json:"email" yaml:"email"`
-	FirstName             string    `json:"firstName" yaml:"firstName"`
-	LastName              string    `json:"lastName" yaml:"lastName"`
-	RegistrationCompleted bool      `json:"registrationCompleted" yaml:"registrationCompleted"`
```

Comment:
What's `RegistrationCompleted` being replaced with?

Reply:
A row in the `users` table with a foreign key reference to this `InternalUser`